### PR TITLE
sidebar responsiveness

### PR DESCRIPTION
**Pull request by GilbertUp**
### What does this PR does?
- This PR responsive side-bar and it has Nav bar!
## How can i test this PR?
clone this [repo](https://github.com/atlp-rwanda/atlp-devpulse-fn.git)
-run run command git checkout ft-side-bar-responsiveness
- run npm i and finally npm run start to start the server.
- 
Screen Shot 2022-10-28 at 
<img width=
13 19 32" src="https://user-images.githubusercontent.com/98816131/198575699-7420f1d4-5357-473e-b17f-b8f78a53afcf.png">
Screen Shot 2022-10-28 at 13 19 24
Screen Shot 2022-10-28 at 13 19 15

### CODE REVIEW

**Review by tysepa — CHANGES_REQUESTED** [review]
move that conflicts and makes sure your code is obvious in other to remove unused code like the one you put in the comment

**Review by tysepa — APPROVED** [review]
looks good

File: src/components/profileDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        tabIndex={0}
+        onClick={handleShowProfileDropdown}
+        onKeyDown={handleShowProfileDropdown}
+        data-testid="keyClick"
+      />
+      <div className="absolute top-[60px] right-0 left-0 ml-auto px-2 md:mx-0 md:left-auto md:right-[10px] z-2  w-full max-w-[220px] h-[calc(100%-70px)]">
+        <div className="flex flex-col flex-nowrap w-full h-max max-h-full bg-white shadow-xl border  dark:border-0 dark:bg-dark-tertiary rounded-[20px]">
+          <div className="flex flex-col w-full p-3 border-border-dark dark:border-white border-b-[0.5px]">
+            {/* <p className="font-bold dark:text-white">{user?.name}</p> */}
+            {/* <p className="font-bold text-sm text-gray-700 dark:text-gray-300">
```
Comment (capitale250) [inline src/components/profileDropdown.tsx:28]: commented codes

Comment (capitale250) [inline src/components/profileDropdown.tsx:28]: commented codes

File: webpack.config.js [hunk 2]
```diff
@@ -23,33 +20,14 @@ module.exports = () => {
...
-        http: require.resolve('stream-http'),
-        https: require.resolve('https-browserify'),
-        os: require.resolve('os-browserify/browser'),
-        path: require.resolve('path-browserify'),
-        punycode: require.resolve('punycode'),
-        process: require.resolve('process/browser'),
-        querystring: require.resolve('querystring-es3'),
-        stream: require.resolve('stream-browserify'),
-        string_decoder: require.resolve('string_decoder'),
-        sys: require.resolve('util'),
```
Comment (capitale250) [inline webpack.config.js:42]: are we loosing all this

**Review by capitale250 — CHANGES_REQUESTED** [review]
squash the commits

**Review by capitale250 — CHANGES_REQUESTED** [review]
squash the commits

**Pull request merged**